src/ddmd/backend/gloop.c

```diff
@@ -794,9 +795,31 @@ void loopopt()
     /* one first, thus moving LIs out as far as possible.       */
 
     doflow = TRUE;                      /* do flow analysis             */
+
+    if (go.mfoptim & MFtime)
+    {
+        cmes("Starting loop unrolling\n");
+        for (loop *l = startloop; l; l = ln)
+        {
+            ln = l->Lnext;
+            if (loopunroll(l))
+            {
+                compdfo();                      // compute depth-first order
+                blockinit();
+                compdom();
+                findloops(&startloop);          // recompute block info
+                doflow = TRUE;
+                if (ln)
+                {   ln = startloop;         // start over
```

Comment:
since you have no code following braces in the code introduced elsewhere, this is jarring